templates/marketplace/dsm-mp.template

```diff
@@ -251,6 +251,8 @@ Mappings:
     sa-east-1:
       PerHost: ami-1f275373
       BYOL: ami-3d057151
+    us-gov-west-1:
+      BYOL: ami-f5de5e94
```

Comment:
Both, it will fail the check in per host template and it's documented in the deployment guide.